client/src/styles/userforms-cms.scss

```diff
@@ -78,8 +78,9 @@
         padding-bottom: 0;
       }
 
-      + .ss-gridfield-item[data-class="SilverStripe\\UserForms\\Model\\EditableFormField\\EditableFieldGroup"] td {
-         border-top: 3px solid $gray-200;
+      + .ss-gridfield-item[data-class="SilverStripe\\UserForms\\Model\\EditableFormField\\EditableFieldGroup"]
+        td {
```

Comment:
This is a little misleading since it’s actually the same selector, can we keep it on one line?

Reply:
Sure. Was an in-editor linter that made this change :/